cedar-lean/Cedar/Data/Map.lean

```diff
@@ -33,31 +33,28 @@ deriving Repr, DecidableEq, Repr, Inhabited
 
 namespace Map
 
-def kvs {α : Type u} {β : Type v} : Map α β → List (Prod α β)
-| .mk kvs => kvs
-
-
-open Except
+abbrev kvs  {α : Type u} {β : Type v} (m : Map α β) := m.1
 
 ----- Definitions -----
 
--- produces an ordered and duplicate free list provided the given map is well formed
-def toList {α : Type u} {β : Type v} (m : Map α β) : List (Prod α β) := m.kvs
-
-/- Creates a well-formed map from the given list of pairs. -/
+/-- Creates a well-formed map from the given list of pairs. -/
 def make {α β} [LT α] [DecidableLT α] (kvs : List (α × β)) : Map α β :=
   Map.mk (kvs.canonicalize Prod.fst)
 
 /-- Empty map -/
 def empty {α β} : Map α β := .mk []
 
+/-- Returns an ordered and duplicate free list provided the given map is well-formed. -/
+def toList {α : Type u} {β : Type v} (m : Map α β) : List (Prod α β) := m.kvs
```

Comment:
Elsewhere we use `List (α × β)` instead of `Prod α β`

Reply:
Yes, I also prefer `α × β` to `Prod` and have been changing it whenever I see it.  Missed this one :)